objectcomputing/check-ins
# Bugfix 2116/email export mishandles commas

Re-implemented member export service. Removed react-csv dependency from web-ui.

To test:
- Log in as admin
- Go to the Send Email page and export members (try ones with commas in titles or locations)
- Go to the profile icon in the top right and select "Download Members"
- In both of the above cases, commas should be handled correctly by wrapping those entries in quotes

Closes #2116
